modules/cluster-logging-loki-deploy.adoc

```diff
@@ -33,7 +33,7 @@ endif::[]
 
 .. Under *Installed Namespace*, select *openshift-operators-redhat*.
 +
```

Comment:
@libander if the namespace contains Community Operators, is there an action a customer can or should take to avoid the conflict the conflict ?